docs/ruleset_functions/get_zone_eflh.md

```diff
@@ -0,0 +1,43 @@
+# get_zone_eflh
+
+**Description:** provides the equivalent full load hours of the zone.  Equivalent full load hours are defined as: any hour where the occupancy fraction is greater than 5% AND the HVAC system is in occupied mode.  For this function, we are recognizing the HVAC system as being in occupied mode if ANY of the HVAC systems serving the zone are in occupied mode.
+**Applicability Note:** hvac_system.fan_system.operation_schedule is being used as the HVAC operation schedule.  Therefore, this check will not work for radiant systems or other systems that do not include a fan
```

Comment:
I am not clear what this functions is for. Let's discuss at the call. Also, given that zones may have multiple spaces and occupancy is defined at space level, should 5% occupancy criteria apply at the space level or zone level? Have not reviewed the function.

Reply:
the logic is based on what is outlined in this [PNNL document](https://www.pnnl.gov/main/publications/external/technical_reports/PNNL-26917.pdf IN THE COMMENTS)